src/components/TicketsTable.tsx

```diff
@@ -28,6 +30,14 @@ function CreatedAtTemplate({ created_at, id }) {
   );
 }
 
+function TeamMemberView({ id, team_member }: ServiceRequestSummary) {
+  return (
+    <Link className="text-gray-900" href={`?ticket=${id}`} key={id}>
```

Comment:
I thought we did away with <links> and had a single onclick for each row, is this code just outdated?

Reply:
I probably brought this back in while merging. It was a complex merge and I missed this, and maybe more. Will fix it